mmv1/third_party/terraform/services/alloydb/resource_alloydb_cluster_test.go

```diff
@@ -50,6 +50,7 @@ resource "google_alloydb_cluster" "default" {
   cluster_id = "tf-test-alloydb-cluster%{random_suffix}"
   location   = "us-central1"
   network    = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}"
+  cluster_type = "PRIMARY"
```

Comment:
Same as on the samples- given you have set a default value on the field, please remove this. Extensive modifications to existing tests to support a change are generally indicative of a regression in the resource, so we generally avoid them.

Reply:
Removed cluster_type field from the existing tests. Default value is set to PRIMARY